spec/controllers/users/registrations_controller_spec.rb

```diff
@@ -384,6 +394,13 @@
       expect(subject).to receive(:ab_finished).with(:demo)
       get :new
     end
+
+    it 'tracks the pageview' do
```

Comment:
Could we track pageviews with a before_action instead to catch them all and avoid a specific test for each page?